internal/user/user_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package user_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestUser(t *testing.T) {
```

Comment:
why this package is using some runner func?
cringe. Default lib is pretty fine btw

Reply:
basic ginkgo syntax, I love ginkgo